### PR TITLE
Fix stuck and timeout issues when exception happens during testing signWithMultiSig.js

### PR DESCRIPTION
**Pull request by aeharvlee**
## Proposed changes

This PR fix stuck and timeout issues when testing `test/signWithMultiSig` with account which have KLAY less than 10.

Stuck happens at `await caver.klay.sendTransaction(txObject)` because there is no exception-handling logic when using async/await function.
I created simple example which can simulate similar problem described above. You can checkout below example.
Example: [Without catch exception, it just wait for timed out](https://gist.github.com/aeharvlee/7dd62207c8bc3728204021d3bbf8c20d)

This makes testing `test/signWithMultiSig` consumes maximum timeout.
You can re-produce this stuck issue with `$ privateKey='0x5677c4adaf9b6fd0c8765ccf1fbe9f1804f2ac9761ee3ef714cca631c74139a6' mocha test/signWithMultiSig.js`)
('0x5677c4adaf9b6fd0c8765ccf1fbe9f1804f2ac9761ee3ef714cca631c74139a6' is just a dummy private key, so you can use it freely.)

It should throw exception right away when exception happens instead of waiting for timed out.
Fixing this issue can provide a more comfortable testing experience for `caver-js` users.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: test/signWithMultiSig.js [hunk 1]
```diff
@@ -59,10 +63,14 @@ const createMultiSigAccount = async () => {
...
-    return caver.klay.sendTransaction(accountUpdateObject)
+    try {
+        await caver.klay.sendTransaction(accountUpdateObject)
+    } catch (error) {
+        throw error
+    }
 }
 
-before(function(done) {
+before(async function() {
```
Comment (jimni1222) [inline test/signWithMultiSig.js:73]: How about changing only before function like below?

```
before(async function() {
   ...
   await createMultiSigAccount()
})
```

I tested test code wont stuck with this change. (without adding `try-catch`)

**Comment by jimni1222** [comment]
> [CLA assistant check](https://cla-assistant.io/klaytn/caver-js?pullRequest=453)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/klaytn/caver-js?pullRequest=453) before we can accept your contribution.
> 
> **Denver.Lee** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/klaytn/caver-js?pullRequest=453) it.

@aeharvlee Please sign the CLA!

File: test/signWithMultiSig.js (hunk 1 above)
Comment (aeharvlee) [reply test/signWithMultiSig.js:73]: @jimni1222 
Thanks for your feed back.
That would be great idea.

I'll push the code reflecting your feedback on my forked branch `fix/stuck-at-exception`

**Comment by aeharvlee** [comment]
> > [CLA assistant check](https://cla-assistant.io/klaytn/caver-js?pullRequest=453)
> > Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/klaytn/caver-js?pullRequest=453) before we can accept your contribution.
> > **Denver.Lee** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> > You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/klaytn/caver-js?pullRequest=453) it.
> 
> @aeharvlee Please sign the CLA!

To make CLA works, I need to change author info on my commit logs.
I'll close this PR and re-produce PR with right author info.

Thank you

**Pull request closed without merging**